libraries/report-size-deltas/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3-slim AS build-env
+ADD . /app
+WORKDIR /app
+
+FROM gcr.io/distroless/python3
+COPY --from=build-env /app /app
+WORKDIR /app
+CMD ["/app/reportsizedeltas.py"]
```

Comment:
Better to do something like this here

```suggestion
FROM python:3.8.2

# Copies your code file from your action repository to the filesystem path `/` of the container
COPY reportsizedeltas.py /reportsizedeltas.py
RUN ["chmod", "+x", "reportsizedeltas.py"]

# Code file to execute when the docker container starts up
ENTRYPOINT ["python", "reportsizedeltas.py"]
```

Reply:
I have committed your suggestion: https://github.com/arduino/actions/pull/18/commits/87ca24221952c7e0ac27e3432449bb0c98193ca6

I did need to change the script path used for the entrypoint command from reportsizedeltas.py to `/reportsizedeltas.py` because the suggested path resulted in the error:
```
python: can't open file 'reportsizedeltas.py': [Errno 2] No such file or directory
```
To make review easier, I made that change in a separate commit: https://github.com/arduino/actions/pull/18/commits/fbba9bddbbf442f26794f555e0db6df88bd41ec4

Let me know if you think I should rebase/squash any of these commits before the merge.